src/org/opensolaris/opengrok/index/IndexDatabase.java

```diff
@@ -969,14 +985,19 @@ private void indexDown(File dir, String parent, IndexDownArgs args)
                             }
                         }
 
-                        // If the file was not modified, skip to the next one.
-                        if (uidIter != null && uidIter.term() != null
-                                && uidIter.term().bytesEquals(buid)) {
-                            BytesRef next = uidIter.next(); // keep matching docs
-                            if (next == null) {
-                                uidIter = null;
-                            }
-                            continue;
+                        /**
```

Comment:
Just a silly thing, why are you using javadoc style of comments instead of ordinary ones?

I believe it would be better to use simple comments because:
```
/*
 *
 */
```

Reply:
I am not using Javadoc style of comments, I am using a begin-comment delimiter with two asterisks. NetBeans conveniently auto-formats that, so I'll stick to that.